cogs/penis.py

```diff
@@ -1,3 +1,4 @@
+from pickletools import decimalnl_long
```

Comment:
why are you importing this?

Reply:
on accident i think VSCode threw that in there.